.github/workflows/ci.yml

```diff
@@ -116,6 +121,7 @@ jobs:
         run: sed 's/enforceTimeLimit="true"/enforceTimeLimit="false"/g' phpunit.xml.dist > phpunit.xml
 
       - name: Run tests
+        continue-on-error: ${{ matrix.php-version > 8.1 }}
```

Comment:
not sure if we want to do it?
or it's only temporary, and to be removed before merge?

Reply:
It was temporary, after updating `phpunitgoodpractices/*` it's no longer needed.